utils/build-script-impl

```diff
@@ -2267,11 +2238,12 @@ for host in "${ALL_HOSTS[@]}"; do
                 )
 
                 if [[ $(is_cross_tools_host ${host}) ]] ; then
-                    add_cross_cmake_options ${host} cmake_options
+                    cmake_options+=("${SWIFT_TARGET_CMAKE_OPTIONS[@]}")
 
                     # CROSS_COMPILE_DEPS_PATH is searched for the SQLite3 dependency.
                     cmake_options+=(
                         -DCMAKE_FIND_ROOT_PATH:PATH="${CROSS_COMPILE_DEPS_PATH}"
+                        -DCMAKE_FIND_ROOT_PATH_MODE_PROGRAM=NEVER
```

Comment:
Can you add a comment here explaining why you are making this change.

Reply:
specifically, let me be more specific. I don't think people use CMAKE_FIND_ROOT_PATH, CMAKE_FIND_ROOT_PATH_MODE_PROGRAM very often. It would be good to just explain why that is useful and maybe link to CMake documentation on the variable.

Reply:
Sure.